src/index.js

```diff
@@ -1,39 +1,24 @@
 import './style.css';
+import './add-rmv-edit.js';
 
 const todoList = document.getElementById('todo-list');
-const lists = [
-  {
-    index: 1,
-    description: 'Wash clothes',
-    completed: true,
-  },
-  {
-    index: 2,
-    description: 'Cook dinner',
-    completed: false,
-  },
-  {
-    index: 3,
-    description: 'Coding challenge',
-    completed: false,
-  },
-  {
-    index: 4,
-    description: 'TAWG',
-    completed: true,
-  },
-];
```

Comment:
Nice job removing the hardcoded array items, however the array itself should remain and functionality of adding and removing to do items should be done using this array.